be/src/vec/functions/array/function_arrays_overlap.h

```diff
@@ -128,6 +129,87 @@ class FunctionArraysOverlap : public IFunction {
         return make_nullable(std::make_shared<DataTypeUInt8>());
     }
 
+    /**
+     * eval inverted index. we can filter array rows with inverted index iter
+     * array_overlap(array, []) -> array_overlap(array, const value)
```

Comment:
What do you mean by `array_overlap(array, []) -> array_overlap(array, const value)` ?

Reply:
we now just support arrays_overlap with second param is const value